internal/clients/http/sources/sources_client_test.go

```diff
@@ -29,4 +30,28 @@ func TestSourcesClient_GetAuthentication(t *testing.T) {
 		_, err = client.GetAuthentication(ctx, "256144")
 		assert.Error(t, err, "Authentication should not succeed with missing link to Provisioning service")
 	})
+
+	t.Run("source with multiple apps", func(t *testing.T) {
+		testSourceId := "256144"
+
+		mux := http.NewServeMux()
+		mux.HandleFunc(fmt.Sprintf("/sources/%s/authentications", testSourceId), func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
+			w.WriteHeader(http.StatusOK)
+			_, err := io.WriteString(w, `{"data":[{"id":"1","authtype":"access_key_secret_key","username":"JDOEAWSUSER","availability_status":"in_progress","resource_type":"Source","resource_id":"1"},{"id":"2","authtype":"arn","username":"arn:aws:iam::123456789999:role/redhat-cost-management-role-0f60c5c","availability_status":"in_progress","resource_type":"Application","resource_id":"1"},{"id":"3","authtype":"provisioning-arn","username":"arn:aws:iam::123456789999:role/redhat-provisioning-role-2f6d01c","availability_status":"in_progress","resource_type":"Application","resource_id":"2"},{"id":"4","authtype":"cloud-meter-arn","username":"arn:aws:iam::123456789999:role/redhat-cloud-meter-role-6331a17","availability_status":"in_progress","resource_type":"Application","resource_id":"3"}],"meta":{"count":4,"limit":100,"offset":0},"links":{"first":"/api/sources/v3.1/sources/1/authentications?limit=100\u0026offset=0","last":"/api/sources/v3.1/sources/1/authentications?limit=100\u0026offset=100"}}`)
+			require.NoError(t, err, "failed to write http body for stubbed server")
+		})
```

Comment:
Nice, I like how you test this, well done. I am just curious, this is an account that has four sources, three are application types:

* arn
* provisioning-arn
* cloud-meter-arn

What is "arn"? I guess that is the first app that was introduced so it does not have a prefix?

Reply:
Why four sources?
If I understood correctly, this is only one source that is linked to four apps/resources.

Reply:
This is an actual data snippet, with changed account numbers and other specifics, but should be almost real data :)

@lzap: yeah, `arn` = cost-management, exactly as you're saying - the first app introduced in sources, so it got the unprefixed arn :)

@adiabramovitch: exactly, I believe one of the links is to Source itself and then there are three actual apps :)